save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -121,10 +121,11 @@ internal class OrganizationController(
     @ApiResponse(responseCode = "200", description = "Successfully changed ability to create contests.")
     @ApiResponse(responseCode = "403", description = "Could not change ability to create contests due to lack of permission.")
     @ApiResponse(responseCode = "404", description = "Organization with such name was not found.")
+    @Suppress("UnsafeCallOnNullableType")
     fun setAbilityToCreateContest(
         @PathVariable organizationName: String,
         @RequestParam isAbleToCreateContests: Boolean,
-        authentication: Authentication
+        authentication: Authentication?,
```

Comment:
I don't think nullable parameter is needed here, because this method already has `PreAuthorize` which would filter out unauthorized requests

Reply:
Oh sure, will fix all those moments